setup.cfg

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, C0103, R0913, E1103, W0142, E0611, E1101, C0330, C0326, W504
+max-line-length = 88
+max-complexity = 18
+select = B, C, E, F, W, T4, B9
+exclude =
+    doc/conf.py
```

Comment:
Can this not go in the toml file also?

Reply:
Unfortunately, it appears `flake8` still doesn't support `pyproject.toml`: https://gitlab.com/pycqa/flake8/-/issues/428

Reply:
Well that's a shame.